themes/edx.org/lms/templates/header.html

```diff
@@ -69,10 +69,9 @@ <h2 class="course-header">
 
     % if user.is_authenticated():
       % if not course or disable_courseware_header:
-        % if not nav_hidden or show_program_listing:
           <nav aria-label="Main" class="nav-main">
             <ul class="left list-inline nav-global authenticated">
-              % if not nav_hidden:
+              % if not nav_hidden and not show_program_listing:
```

Comment:
When you bring in the top level if statement, this line does make sense. However, the logic on line 87 is implicitly changed. Should we also include "not nav_hidden" block on line 87 as well?